ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -413,6 +414,32 @@ svi_profiles:
             weight: < 0-65535>
             bfd: < true | false >
 
+        # Dictionary of BGP peer groups definitions | Optional.
+        # This will configure BGP peer groups to be used inside the tenant VRF for peering with external devices.
+        # Since BGP peer groups are configured at higher BGP level, shared between VRFs,
+        # peer_group names should not overlap between VRFs..
+        bgp_peer_groups:
+          - name: < BGP peer group name >
+            remote_as: < bgp_as >
+            description: "< description as string >"
+            send_community: < standard | extended | large | all >
+            next_hop_self: < true | false >
+            maximum_routes: < 0-4294967294 >
+            default_originate:
+              enabled: < true | false >
+              always: < true | false >
+            # Nodes is required to restrict configuration of BGP peer groups to certain nodes in the network.
+            nodes: [ < node_1 >, < node_2> ]
+            update_source: < interface >
+            bfd: < true | false >
+            ebgp_multihop: < 1-255 >
+            # Next hop settings can be either ipv4 or ipv6 for one neighbor, this will be applied by a uniquely generated route-map per neighbor.
+            # Next hop takes precedence over route_map_out.
+            set_ipv4_next_hop: < IPv4_address >
+            set_ipv6_next_hop: < IPv6_address >
+            route_map_out: < route-map name >
+            route_map_in: < route-map name >
+            local_as: < local BGP ASN >
```

Comment:
Can we also add this structure to the tenant level (not per vrf)?  Use-case is the following:
I have pair of Border Leafs connected to External routers. BLs peers with External routers in multiple VRFs over l3 subifs. Route-map and peer-group for all those multiple BGP peerings for each VRF will be same (single peer-group and single set of RMs in/out). With suggested data model I will need to repeat peer-group in each VRF or define it once in first VRF, which is a bit ugly.

Reply:
I like this suggestion, but let's do it in a separate PR and get this one merged.
@lermilov @xaviramon